msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -215,6 +223,309 @@ async def on_members_added_activity(
 
 Messages sent between users and bots include internal channel data within the message. This data allows the bot to communicate properly on that channel. The Bot Builder SDK allows you to modify the message structure.
 
+## Update message
+
+When you edit or Undo delete a message sent to a bot in a chat, the bot gets a notification of the edit or undo delete an event.
```

Comment:
"When you edit or Undo delete a message sent to a bot in a chat, the bot gets a notification of the edit or undo delete an event."

Could you update it to

"When you edit or undelete a message in a chat, the bot gets a notification of the edit message or undelete message event."

Reply:
updated as suggested